recipes/sentry-breakpad/all/conanfile.py

```diff
@@ -38,7 +38,7 @@ def requirements(self):
 
     def validate(self):
         if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, 11)
+            tools.check_min_cppstd(self, tools.Version(self.version) < "0.5.4" ? 11 : 17)
```

Comment:
Oops :) Python's way is:

```suggestion
            tools.check_min_cppstd(self, 11 if tools.Version(self.version) < "0.5.4" else 17)
```